config.toml

```diff
@@ -60,7 +60,7 @@ google_analitycs_id = "" # Your ID
 # copyright
 copyright = "&copy; 2020 Sauce Labs Inc., all rights reserved. SAUCE and SAUCE LABS are registered trademarks owned by Sauce Labs Inc. in the United States, EU, and may be registered in other jurisdictions."
 # social links on hero
-enable_social_links_hero = false
+enable_social_links_hero = true
```

Comment:
Why is this needed?
This is different from what the designer recommended. Now we have a big red gap between the hero image and the first section.

Reply:
@diemol fixed